src/Framework/MockObject/MockObject.php

```diff
@@ -28,4 +28,8 @@ public function __phpunit_hasMatchers(): bool;
     public function __phpunit_setReturnValueGeneration(bool $returnValueGeneration): void;
 
     public function expects(Invocation $matcher): BuilderInvocationMocker;
+
+    // @TODO get rid of this - required by psalm-only right now - bug in parsing @method?
+    // @TODO this addition is a BC break and needs to be reverted before merge.
+    public function method($constraint) : BuilderInvocationMocker;
```

Comment:
Need https://github.com/vimeo/psalm/issues/1692 to get rid of this

Reply:
Done